requirements-feature.txt

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+# This file is adding constraints to be used when a PR is targeting a feature/* branch
+-e git+https://github.com/inveniosoftware/invenio-records.git@feature/custom-fields#egg=invenio-records
```

Comment:
to be removed (and CI fix too) before merging and after release of these packages?